src/services/data-fabric/index.ts

```diff
@@ -1 +1,21 @@
-export * from './entities';
+/**
+ * Data Fabric Services Module
+ *
+ * Provides access to UiPath Data Fabric services for entity management.
+ * @module
+ */
+
+// Export service with cleaner name and keep EntityService for legacy UiPath class
+export { EntityService as Entities, EntityService } from './entities';
+
+// Re-export types for convenience
+export type * from '../../models/data-fabric/entities.types';
```

Comment:
Same here - we should use decorators instead of relative paths
```suggestion
export type * from '@models/data-fabric/entities.types';
```